MonoGame.Framework/WindowsUniversal/UAPGamePlatform.cs

```diff
@@ -132,13 +132,22 @@ public override void RunLoop()
 
         public override void StartRunLoop()
         {
-            CompositionTarget.Rendering += (o, a) =>
+            CoreWindow.GetForCurrentThread().Dispatcher.RunIdleAsync((a) =>
             {
-				UAPGameWindow.Instance.Tick();
-                GamePad.Back = false;
-            };
+                CoreWindow.GetForCurrentThread().Dispatcher.RunAsync( CoreDispatcherPriority.Low, OnRenderFrame);
+            });
```

Comment:
It's too early to call RenderFrame(). And I had to dispatch with  RunIdleAsync the first time so that XAML initialize & create events arrives before OnRenderFrame();.